packages/vscode-extension/package.nls.json

```diff
@@ -368,6 +368,7 @@
     "teamstoolkit.guides.addTab.label": "Configure Tab Capability",
     "teamstoolkit.guides.cardActionResponse.detail": "Automate repetitive workflows for common business processes in conversation",
     "teamstoolkit.guides.cardActionResponse.label": "Initiate Sequential Workflows in Teams",
+    "teamstoolkit.guides.notificationBot.label": "Overview of the Notification bot template",
```

Comment:
Capitalize first letter to 'Overview of the Notification Bot Template'?

Reply:
Thanks, updated.